pkg/daemon/daemon.go

```diff
@@ -76,6 +80,8 @@ func (dn *Daemon) Run(stopCh <-chan struct{}, exitCh <-chan error) error {
 	glog.V(0).Info("Run(): start daemon")
 	// Only watch own SriovNetworkNodeState CR
 
+	tryEnableRdma()
```

Comment:
There is no check if it failed or passed?

Reply:
If it failed, it just means this host is RDMA incapable, this daemon shall keep running. But maybe we shall add a field in status of SriovNetworkNodeState to indicate whether host is RDMA capable or not.